packages/x-date-pickers-pro/src/internal/hooks/useRangePosition.ts

```diff
@@ -0,0 +1,43 @@
+import useControlled from '@mui/utils/useControlled';
+import useEventCallback from '@mui/utils/useEventCallback';
+import { RangePosition } from '../models/range';
+
+export interface UseRangePositionProps {
+  /**
+   * The position in the range (start or end date) currently edited.
```

Comment:
What do you think about trying not to repeat the option names in the comment? 🤔 

```suggestion
   * The position in the currently edited date range.
```